scan/src/bindings/StdMsgRequest.re

```diff
@@ -0,0 +1,35 @@
+type amount_t = {
+  amount: string,
+  denom: string,
+};
+
+type send_value_t = {
+  oracleScriptID: string,
+  calldata: string,
+  requestedValidatorCount: string,
+  sufficientValidatorCount: string,
+  expiration: string,
+  prepareGas: string,
+  executeGas: string,
+  sender: string,
+};
+
+type msg_t = {
+  [@bs.as "type"]
+  type_: string,
+  value: send_value_t,
+};
+
+type fee_t = {
+  amount: array(amount_t),
+  gas: string,
+};
+
+type t = {
```

Comment:
Let's create a wrapper function that helps creating `t` from known types (`Address.t`, `int`, etc) so the caller is safe.